concepts/throttling.md

```diff
@@ -0,0 +1,39 @@
+# Microsoft Graph throttling guidance
+
+## What is throttling?
+
+Throttling limits the number of concurrent calls to a service to prevent overuse of resources. Microsoft Graph is designed to handle a very high volume of requests. In the event of an overwhelming number of requests, throttling helps maintain optimal performance and reliability of the Microsoft Graph service.
+
+Throttling limits vary based on the scenario. For example, if you are performing a large volume of writes to your tenant, the possibility for throttling is higher than if you are only performing reads.
```

Comment:
I would say to "a tenant".  This would then apply for single tenant and multi-tenant app developers.